app/templates/notifications/user-research-consent.html

```diff
@@ -0,0 +1,72 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}User reserch notification – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": dashboard_url,
+        "label": "Your account"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="grid-row notification-user-research">
+
+  <div class="column-two-thirds">
+    {% with heading = "Join the user research mailing list" %}
+      {% include "toolkit/page-heading.html" %}
+    {% endwith %}
+    
+    <p class="text">
+      Help us improve the Digital Marketplace by taking part in user research.
+    </p>
+    <p class="text">
+      You don’t have to take part just because you’ve signed up and you can unsubscribe at any time.
+    </p>
+
+    <form action="{{ url_for('main.user_research_consent') }}" method="POST">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+      {%
+        with
+        type = "checkbox",
+        name = "user_research_opt_in",
+        value = [userResearchOptIn],
+        options = [
+          { 
+            "label": "Send me emails about opportunities to get involved in user research",
+            "value": True,
+          }
+        ]
+      %}
+        {% include "toolkit/forms/selection-buttons.html" %}
+      {% endwith %}
+
+      {% with label="Save changes", type="save" %}
```

Comment:
This button should get a `name` argument so we can find it with the functional tests